python/mxnet/foo/nn/layer.py

```diff
@@ -207,6 +222,80 @@ def generic_forward(self, F, x, *args, **kwargs):
         raise NotImplementedError
 
 
+class _Binary(Layer):
```

Comment:
I don't think Add/sub are good names. I would expect Add to add two NDArrays, not split then merge two layers.

The documentation also needs improvement.